documentation/src/object-Initializer.md

```diff
@@ -0,0 +1,14 @@
+# Initializer
+
+## Index Initializers
+
+Use index initializers with dictionary.
+
+```csharp
+var dict = new Dictionary<string, int>
```

Comment:
Nitpick: You could also include a negative example of what NOT to do. We do this in other places as well.

Reply:
a307d749dc25dd00af008083a96e8e6b9891d827